.gitignore

```diff
@@ -1,2 +1,8 @@
 target/
 /bin/
+/test-output/
+src/test/resources/molgenis01.properties
```

Comment:
I think you can remove 
src/test/resources/molgenis01.properties
src/test/resources/molgenis62.properties

and add ignore them only on your machine
add them to them locally into .git/info/exclude